app/enums/AuditType/AuditType.scala

```diff
@@ -121,3 +121,7 @@ case object CreditsSummaryResponse extends AuditType {
 case object MangeIncomeSourcesHome extends AuditType {
   override val name: String = "ViewAndManageIncomeSourcesHome"
 }
+
+case object Obligations extends AuditType {
+  override val name: String = "Obligations"
```

Comment:
its seems quite short name for the auditType, might be something you need to raise with the BAs; but its is correct with respect what is specified in the story

Reply:
I think the idea is to share this event between all of the obligations pages, journey and incomeSourceType get specified in the details